Problem with Prettier

After the last pull I am getting an error that there is a problem with prettier configuration `["INFO" - 7:39:20 PM] Extension Name: "prettier-vscode".
["INFO" - 7:39:20 PM] Extension Version: "3.20.0".
["INFO" - 7:39:20 PM] Enabling prettier for languages
[
  "css",
  "graphql",
  "html",
  "javascript",
  "javascriptreact",
  "json",
  "json5",
  "jsonc",
  "less",
  "markdown",
  "mdx",
  "mongo",
  "postcss",
  "scss",
  "typescript",
  "typescriptreact",
  "vue",
  "yaml"
]
["INFO" - 7:39:20 PM] Enabling prettier for range supported languages
[
  "graphql",
  "javascript",
  "javascriptreact",
  "json",
  "typescript",
  "typescriptreact"
]
["INFO" - 7:39:30 PM] Formatting /Users/adham/Work/corona/orginal/Corona-tracker/client/src/components/Table.js
["ERROR" - 7:39:30 PM] Invalid prettier configuration file detected.
["ERROR" - 7:39:30 PM] Cannot find module '@blockstack/prettier-config' from '/Users/adham/Work/corona/orginal/Corona-tracker/client'
Error: Cannot find module '@blockstack/prettier-config' from '/Users/adham/Work/corona/orginal/Corona-tracker/client'
	at Function.sync (/Users/adham/.vscode/extensions/esbenp.prettier-vscode-3.20.0/node_modules/prettier/index.js:15946:13)
	at Object.transform (/Users/adham/.vscode/extensions/esbenp.prettier-vscode-3.20.0/node_modules/prettier/index.js:22820:36)
	at /Users/adham/.vscode/extensions/esbenp.prettier-vscode-3.20.0/node_modules/prettier/third-party.js:4550:32
	at async Promise.all (index 0)
["INFO" - 7:40:19 PM] Formatting /Users/adham/Work/corona/orginal/Corona-tracker/client/src/components/Table.js
["ERROR" - 7:40:19 PM] Invalid prettier configuration file detected.
["ERROR" - 7:40:19 PM] Cannot find module '@blockstack/prettier-config' from '/Users/adham/Work/corona/orginal/Corona-tracker/client'
Error: Cannot find module '@blockstack/prettier-config' from '/Users/adham/Work/corona/orginal/Corona-tracker/client'
	at Function.sync (/Users/adham/.vscode/extensions/esbenp.prettier-vscode-3.20.0/node_modules/prettier/index.js:15946:13)
	at Object.transform (/Users/adham/.vscode/extensions/esbenp.prettier-vscode-3.20.0/node_modules/prettier/index.js:22820:36)
	at /Users/adham/.vscode/extensions/esbenp.prettier-vscode-3.20.0/node_modules/prettier/third-party.js:4550:32
	at async Promise.all (index 0)
`

--- a/client/package.json
+++ b/client/package.json
@@ -33,9 +33,6 @@
     "redux-thunk": "^2.3.0",
     "chart.js": "^2.9.3"
   },
-  "devDependencies": {
-    "@blockstack/prettier-config": "0.0.5"
-  },
   "scripts": {
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
@@ -58,6 +55,7 @@
     ]
   },
   "devDependencies": {
+    "@blockstack/prettier-config": "0.0.5",
     "@babel/core": "^7.8.7",
     "babel-jest": "^24.9.0",
     "canvas": "^2.6.1",

